rasa_core/policies/embedding_policy.py

```diff
@@ -74,6 +74,9 @@ class EmbeddingPolicy(Policy):
         "batch_size": [8, 32],
         # number of epochs
         "epochs": 1,
+        # set random seed to any int to get reproducible results
+        # try to change to another int if you are not getting good results
```

Comment:
Let's rephrase it. Because the main reason for bad results is probably not the random seed